src/widgets/text.rs

```diff
@@ -28,6 +28,20 @@ pub struct TextProps {
     pub focusable: Option<bool>,
 }
 
+impl Default for TextProps {
+    fn default() -> Self {
+        TextProps {
+            content: "".to_string(),
+            font: Some(bevy_kayak_ui::DEFAULT_FONT.into()),
```

Comment:
This actually won't work since `bevy_kayak_ui` is supposed to only be accessible with the `bevy_renderer` feature enabled.

But I'm looking into doing an overhaul of how fonts are handled so it might not be necessarily needed.